doc/deployment/aws_cloudfront.md

```diff
@@ -0,0 +1,78 @@
+# AWS Cloudfront
+
+For a production installation we recommend using cloudfront.
+
+To deploy a production ready AWS cluster:
+
+1) Follow the instructions for a normal AWS deployment
+
+You'll need to use the 'one shot' script. To do that you'll need to clone this repo, cd to it, and run the script:
```

Comment:
There are two subtle things here that are different. We need them to pass an additional flag (`--use-cloudfront`) and we need them to save the output (for a future step), hence the redirection. So I think it's ok to keep it here.